### ISSUE TITLE
Unable to stub child components

### ISSUE THREAD

**Issue by gsccheng** [vuejs/vue-test-utils#969]
### Version
1.0.0-beta.20

### Reproduction link
[https://stackoverflow.com/questions/52304864/unable-to-mount-element-ui-child-component-in-test](https://stackoverflow.com/questions/52304864/unable-to-mount-element-ui-child-component-in-test)

### Steps to reproduce
I have written out the steps to reproduce here. Thanks:

https://stackoverflow.com/questions/52304864/unable-to-mount-element-ui-child-component-in-test

### What is expected?
Able to mount the Vue component being tested

### What is actually happening?
Unable to mount the Vue component seemingly because of a missing child component that is not being stubbed out.

---
Thanks!

**Comment by eddyerburgh** [comment]
Have you tried using the latest beta.25?

**Comment by gsccheng** [comment]
@eddyerburgh Yes I have but figured I reference the beta.20 for my example since that is the version being used by Vue-cli now.

**Comment by gsccheng** [comment]
Looks like it is an Element-UI bug:

https://github.com/ElemeFE/element/issues/12634

**Comment by hengkx** [comment]
I use version `1.0.0-beta.25`.
![image](https://user-images.githubusercontent.com/8358236/48813562-64eb5880-ed72-11e8-9c96-a114ac199d36.png)
remove  code
```js
<el-pagination @size-change="sizeChange" :current-page="searchParams.current" @current-change="getCompanyList" background layout="sizes, prev, pager, next" :total="total" :page-sizes="[10, 20, 50, 100]" :page-size="searchParams.pageSize">
    </el-pagination>
```
![image](https://user-images.githubusercontent.com/8358236/48813619-995f1480-ed72-11e8-9647-434d97754c56.png)

**Comment by eddyerburgh** [comment]
The original issue is because you are not stubbing the el-select component correctly. You need to use the name that the component will attempt to be resolved as. The easiest option is to use the same name as the tag name used to render the component:

```js
mount(TestComponent, {
  stubs: ['el-select']
})
```

The `$el` issue is caused by the synchronous Transition stub component that Vue Test Utils adds by default. You can turn it off with the `stubs` options:

```js
mount(TestComponent, {
  stubs: {
    transition: false
  }
})
```

This will be fixed after Vue 2.5.18 is released.

Closing in favor of #958
